test/unit/api/datatype_api_black.cpp

```diff
@@ -37,6 +37,10 @@ TEST_F(TestApiBlackDatatype, mkDatatypeSort)
   Datatype d = listSort.getDatatype();
   DatatypeConstructor consConstr = d[0];
   DatatypeConstructor nilConstr = d[1];
+  ASSERT_FALSE(d.isNull());
```

Comment:
I would prefer to have a dedicated `isNull` test, similarly to https://github.com/cvc5/cvc5/blob/master/test/unit/api/term_black.cpp#L215